addons/account_voucher/migrations/9.0.1.0/pre-migration.py

```diff
@@ -26,3 +31,15 @@ def migrate(cr, version):
     cr.execute("update account_voucher_line set name='/' where name is null")
     openupgrade.copy_columns(cr, column_copies)
     delete_payment_views(cr)
+
+    cr.execute('SELECT count(*) FROM account_voucher WHERE tax_id IS NOT NULL')
+    taxed_vouchers = cr.fetchone()[0]
```

Comment:
Since we're counting PG should always return a single tuple.  The following shows it:

```
tmp=# CREATE TABLE tmp(name VARCHAR);
CREATE TABLE
tmp=# select count(*) from tmp where name IS NOT NULL;
 count 
-------
     0
(1 row)

tmp=# select count(*) from tmp where name IS NULL;
 count 
-------
     0
(1 row)
```